### ISSUE TITLE
Incomplete rendering 

### ISSUE THREAD

**Issue by moodymudskipper** [javierluraschi/nomnoml#17]
I had an issue with incomplete rendering, and posted this issue, which was solved : https://github.com/skanaar/nomnoml/issues/128

Would it be possible to update the package so it uses the last version of the library ?

**Comment by moodymudskipper** [comment]
I've found the following ressource showing how to add an active binding to one's package : 

https://stackoverflow.com/questions/54204187/use-makeactivebinding-inside-a-package

This might be an relevant addition to your book.

One thing that I haven't managed to do is document this active binding.

I tried this : 

```
#' var
#'
#' @export
#' @name var
NULL
```

and it works but an ugly warning appears when loading the namespace : 

> previous export '‘var’' is being replaced

To document it on the same page as another function we can use an `@alias`, and enter manually the `@usage` definitions. Which might be an acceptable alternative if there is no clean way to give the active binding its own help page.

**Comment by andrie** [comment]
I'm working on a fix.  In my local version, when I run this example, it works as expected:

```r
library(nomnoml)

diagram <- "
#direction: vertical
[fun(x)]  -> [if (x)]
[if (x)] y -> [if (y)]
[if (y)] y -> [foo]
[foo]  -> [<end> -2]
[if (y)] n -> [bar]
[bar]  -> [<end> -2]
[<end> -2]  -> [<end> -1]
[if (x)] n -> [<end> -1]
"

nomnoml(diagram)
```

![image](https://user-images.githubusercontent.com/479998/101345115-80086280-387e-11eb-909f-0336d7875be6.png)

**Comment by andrie** [comment]
This should now be fixed in the master branch (commit ce8c63a21d7486696d90d19dc0cc7b15cfc40a52).  Would you mind doing some testing and let me know if anything breaks?

**Comment by moodymudskipper** [comment]
Sure, I will, many thanks!
